lib/ObjWriter/jitDebugInfo.h

```diff
@@ -0,0 +1,46 @@
+#ifndef JIT_DEBUG_INFO_H
+#define JIT_DEBUG_INFO_H
+
+typedef unsigned int DWORD;
+#define _TARGET_AMD64_ 1
+
```

Comment:
This is due to conflict on COFF related constant in enumeration in LLVM and macro in winnt.h And also you need to include "globals.h" to make this work.

Let's keep your change as it is for now and we need some discussions.